spec/helpers/getImage.js

```diff
@@ -40,10 +44,6 @@ describe('getImage helper', function() {
                 input: '{{getImage "http://example.com/image.jpg"}}',
                 output: 'http://example.com/image.jpg',
             },
-            {
```

Comment:
removed this as it is duplicative of the test above.

of note, if you pass a URL with 2 query strings, we do escape it - but I think that's kind of an edge case so I don't care about it in this PR.